doc/python-usage.rst

```diff
@@ -1,37 +1,32 @@
 Python usage
 ------------
+LCD displays can be driven with python using the various implementations in the
+:py:mod:`luma.lcd.device` package.  There are several device classes available
+and usage is very simple if you have ever use
+`Pillow <https://pillow.readthedocs.io/en/latest/>`_ of PIL.
 
-Pixel Drivers
-^^^^^^^^^^^^^
-The PCD8544 is driven with python using the implementation in the
-:py:class:`luma.lcd.device.pcd8544` class. Likewise, to drive the ST7735, ST7567
-or UC1701X, use the :py:class:`luma.lcd.device.st7735`, 
-:py:class:`luma.lcd.device.st7567` or :py:class:`luma.lcd.device.uc1701x`
-class respectively. For the ILI9341, use :py:class:`luma.lcd.device.ili9341`.
-Usage is very simple if you have ever used
-`Pillow <https://pillow.readthedocs.io/en/latest/>`_ or PIL.
+To begin you must import the device class you will be using and the interface
+class that you will use to communicate with your device:
 
-First, import and initialise the device:
+In this example, we are using an SPI interface with a pcd8544 display.
 
 .. code:: python
 
-  from luma.core.interface.serial import spi
+  from luma.core.interface.serial import i2c, spi, parallel, pcf8574
   from luma.core.render import canvas
-  from luma.lcd.device import pcd8544, st7735, uc1701x, ili9341
+  from luma.lcd.device import pcd8544, st7735, st7567, uc1701x, ili9341, hd44780
 
   serial = spi(port=0, device=0, gpio_DC=23, gpio_RST=24)
   device = pcd8544(serial)
 
-The display device should now be configured for use. Note, all the example code
-snippets in this section are interchangeable between PCD8544 and ST7735
-devices.
+The display device should now be configured for use.
```

Comment:
configured for use. -> properly configured.

Reply:
Changed as suggested but consistency issue with luma.oled docs.